<launch>
  <arg name="robot_id" default="robot_"/>
  <arg name="frequency" default="10"/>
  <arg name="map_frame" default="$(arg robot_id)map"/>
  <arg name="base_frame" default="$(arg robot_id)base_footprint"/>
  <arg name="topic_republish" default="$(arg robot_id)pose"/>

  <node pkg="pose_publisher" type="pose_publisher" name="node_pose_publisher"  output="screen">
    <param name="publish_frequency" value="$(arg frequency)"/>
    <param name="map_frame" value="$(arg map_frame)"/>
    <param name="base_frame" value="$(arg base_frame)"/>
    <param name="topic_republish" value="$(arg topic_republish)"/>
  </node>
</launch>
